### PR TITLE
perf improvement get contingencies in one request

### PR DESCRIPTION
**Pull request by jamal-khey**
depend on [action server](https://github.com/gridsuite/actions-server/pull/102)

### CODE REVIEW

File: src/main/java/org/gridsuite/securityanalysis/server/service/ActionsService.java [hunk 1]
```diff
@@ -58,4 +58,21 @@ public List<ContingencyInfos> getContingencyList(String name, UUID networkUuid,
 
         return restTemplate.exchange(baseUri + path, HttpMethod.GET, null, new ParameterizedTypeReference<List<ContingencyInfos>>() { }).getBody();
     }
+
```
Comment (antoinebhs) [inline src/main/java/org/gridsuite/securityanalysis/server/service/ActionsService.java:61]: should we remove: public List<ContingencyInfos> getContingencyList(String name, UUID networkUuid, String variantId) { 
as it's only used in test ?
And refactor the tests to use the new method ?

**Review by antoinebhs — CHANGES_REQUESTED** [review]
Remove old method and refactor the tests

File: src/main/java/org/gridsuite/securityanalysis/server/service/ActionsService.java (hunk 1 above)
Comment (jamal-khey) [reply src/main/java/org/gridsuite/securityanalysis/server/service/ActionsService.java]: done

**Review by antoinebhs — APPROVED** [review]

**Review by sBouzols — APPROVED** [review]
Add a simple TU for coverage to test calling function without ids maybe

**Pull request merged**